micrometer-spring-legacy/build.gradle

```diff
@@ -48,10 +48,13 @@ dependencies {
 
     testCompile 'org.springframework.boot:spring-boot-starter-web'
     testCompile 'org.springframework.boot:spring-boot-actuator'
-
-    testCompile 'junit:junit:4.+'
     testCompile 'org.springframework.boot:spring-boot-starter-test'
 
+    // JUnit 5
```

Comment:
Many of the test facilities in Spring Boot 1.5.x assume JUnit 4, which is why all the tests in this module are in JUnit 4 (e.g. `@RunWith(SpringRunner.class)` doesn't have a JUnit 5 corollary in Boot 1.5.x.